seeds/data/json/action.json

```diff
@@ -1,317 +1,317 @@
 [
-  {
+  { "id": 1,
```

Comment:
IDs are explicitly necessary in order to guarantee `PreReq` seed data aligns as intended.